### PR TITLE
implemented changes for ADOP-817

### PR DESCRIPTION
**Pull request by praneethhm**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ADOP-817

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
```

### CODE REVIEW

**Comment by abidfs** [comment]
@praneethhm closing this one, please re-open it tomorrow
we are having issues with preview env pod limits

**Review by sathishkannan-hmcts — CHANGES_REQUESTED** [review]

**Comment by gauravtomar-gh** [comment]
@rajatkgupta29, This is going to PROD, please review.

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -26,4 +26,42 @@
     <notes>spring-*-5.3.18</notes>
     <cve>CVE-2022-22968</cve>
   </suppress>
+  <suppress>
+    <notes>spring-retry-1.3.3</notes>
+    <cve>CVE-2016-1000027</cve>
+    <cve>CVE-2022-22965</cve>
+  </suppress>
+  <!--
```
Comment (rajatkgupta29) [inline config/owasp/suppressions.xml:34]: Remove the commented code

File: src/main/java/uk/gov/hmcts/reform/adoption/Application.java [hunk 2]
```diff
@@ -29,7 +30,8 @@
         CoreCaseDataApi.class,
         CaseAssignmentApi.class,
         DocumentManagementClient.class,
```
[the remark this answers is not in the document]
Comment (rajatkgupta29) [reply src/main/java/uk/gov/hmcts/reform/adoption/Application.java:32]: Yes

**Review by rajatkgupta29 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**